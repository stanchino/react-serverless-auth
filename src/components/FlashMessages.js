import React from "react";

import Message from "./Message";

export default ({ flash: { error, notice } }) => (
    <div className={"flash"}>
        <Message message={error} style={{ color: "red" }} />
        <Message message={notice} style={{ color: "green" }} />
    </div>
);