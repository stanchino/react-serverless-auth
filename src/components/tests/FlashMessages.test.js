import { matchSnapshot } from "./shared-examples";

import FlashMessages from "../FlashMessages";

describe("FlashMessages", () => {
    it("matches the snapshot", () => matchSnapshot(FlashMessages, { flash: { error: "error", notice: "notice" } }));
});