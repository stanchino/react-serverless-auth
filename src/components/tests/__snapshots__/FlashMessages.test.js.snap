// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlashMessages matches the snapshot 1`] = `
<div
  className="flash"
>
  <div
    style={
      Object {
        "color": "red",
      }
    }
  >
    <strong>
      error
    </strong>
  </div>
  <div
    style={
      Object {
        "color": "green",
      }
    }
  >
    <strong>
      notice
    </strong>
  </div>
</div>
`;
